store/store_download.go

```diff
@@ -475,6 +476,10 @@ func downloadImpl(ctx context.Context, name, sha3_384, downloadURL string, user
 			pbar = progress.Null
 		}
 		dlSize = float64(resp.ContentLength)
+		if dlSize == 0 {
+			logger.Noticef("unexpected Content-Length: 0 for %s", downloadURL)
+		}
+		logger.Debugf("Download size for %s: %d", downloadURL, resp.ContentLength)
```

Comment:
I would consider upgrading this Noticef and including the name.

Reply:
I'm not sure, I fear this is too much noise, we're mostly interested in pathological case (but it is unclear if it's even possible).

Reply:
I think given our recent problems with downloads stalling or getting stuck, making more info visible from the logs is a net good thing short-term, even if it is too noisy in most cases it may help us eventually crack a subtle bug. I'm +1 for this being a Noticef

Reply:
why appear case if there rest is making the logs lowercare?